CONTRIBUTING.md

```diff
@@ -519,22 +505,22 @@ https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/maintain
 
    ```sh-session
    mpdev publish \
-         --deployer_image=gcr.io/YOUR-PUBLIC-PROJECT/COMPANY-IDENTIFIER/YOUR-APP/deployer:VERSION \
-         --gcs_repo=gs://YOUR-BUCKET/COMPANY-IDENTIFIER/YOUR-APP:TRACK
+         --deployer_image=gcr.io/YOUR-PUBLIC-PROJECT/COMPANY-IDENTIFIER/deployer:VERSION \
+         --gcs_repo=gs://YOUR-BUCKET/COMPANY-IDENTIFIER:TRACK
    ```
 
    For example:
 
    ```sh-session
    mpdev publish \
-         --deployer_image=gcr.io/conjur-cloud-launcher-onboard/cyberark/conjur-open-source/deployer:1.6.1 \
-         --gcs_repo=gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark/conjur-open-source:1.6
+         --deployer_image=gcr.io/conjur-cloud-launcher-onboard/cyberark/deployer:1.6.1 \
+         --gcs_repo=gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark:1.6
```

Comment:
is this correct? maybe we should just have `{version}` instead of 1.6 for future releases?

Reply:
Right, we can use {version} here in place of the `1.6.1`. For the `1.6` (track version), I'll add a bash step to derive that from the patch version.